app/controllers/Users/registrations_controller.rb

```diff
@@ -60,7 +60,7 @@ def copy_file_contents(file_name,user)
     end
 
   def delete_user_file
-    File.delete("public/users/#{current_user.id}.js")
+    File.delete("public/users/#{current_user.id}.min.js")
```

Comment:
File.delete("public/users/#{current_user.id}.min.js") while deleting , you are deleting min.js and 
while creation , file name is only .js